### PR TITLE
Updates to 2019 roadmap

### PR DESCRIPTION
**Pull request by luisivan**
- Marks as done or coming soon items that we have already completed or started/about to start
- Adds new items to the scope
- Removes other items

Review needed on items that I think we should remove (see comments directly on the PR)

### CODE REVIEW

File: teams/Aragon One/2019.md [hunk 1]
```diff
@@ -44,12 +44,9 @@ The App Center will allow users to discover, get and manage apps developed by Ar
 
 Focus areas for this initiative:
 
-- **Upgrading apps**: Provide users with a secure way to install new app versions, fixes for features or enhancements to already installed/in use apps.
-
+- ✅ **Upgrading apps**: Provide users with a secure way to install new app versions, fixes for features or enhancements to already installed/in use apps.
 - **Browsing, installing & uninstalling apps**: Enhance app discovery and app management for end users.
 
-- **Monetization/incentivisation model for app developers**: Provide app developers with different monetisation models to create revenue from the apps they publish in the Aragon App Center.
-
```
Comment (luisivan) [inline teams/Aragon One/2019.md:52]: I think we should take this off, and the reason is simple: first we need users, before the platform becomes interesting for devs

File: teams/Aragon One/2019.md [hunk 2]
```diff
@@ -122,9 +117,44 @@ The Aragon Court is a service protocol designed to incentivize honest arbitratio
...
+### 🔜 I11 – Aragon client re-design and general enhancements
+
+We plan to re-design the Aragon client by creating a new Aragon Human Interface Guidelines that standardizes how to create Aragon apps.
+
+As part of this re-design, we will also re-design the following items, based on user feedback:
+
+- The Permissions app
+- The Voting app
+
+### 🔜 I12 – Support Aragon Funraising development
```
Comment (sohkai) [inline teams/Aragon One/2019.md]:
```suggestion
### 🔜 I12 – Support Aragon Fundraising development
```

Comment (izqui) [inline teams/Aragon One/2019.md]:
```suggestion
### 🔜 I12 – Support Aragon Fundraising development
```

File: teams/Aragon One/2019.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ The goal is to build an identity experience that is seamless and allows individu
 
 Focus areas for this initiative:
 
-- **Individual identity**: Allow individuals to create and manage their user profile, mapping their address to a human readable ENS name that can be used to interact with apps within the organization.
+- ✅ **Individual identity**: Allow individuals to create and manage their user profile, mapping their address to a human readable ENS name that can be used to interact with apps within the organization.
```
Comment (sohkai) [inline teams/Aragon One/2019.md]: I would argue this is not complete, but as of now we're shifting to supporting Autark/OWL's efforts on this.

File: teams/Aragon One/2019.md [hunk 4]
```diff
@@ -249,68 +275,50 @@ The team is fully remote and distributed across the world, primarily in Europe a
...
   - Delfi: [GitHub](https://github.com/delfipolito)
   - Gorka: [GitHub](https://github.com/AquiGorka)
-  - Paul: [GitHub](https://github.com/drcmda)
-
-- Product Management and UX: 1
-
-  - Paty: [Github](https://github.com/dizzypaty), [Twitter](https://twitter.com/dizzypaty), [website](http://patydavila.com/)
-
+  - Facu: [GitHub](https://github.com/facuspagnuolo)
+- Daniel: [GitHub](https://github.com/2color)
```
Comment (izqui) [inline teams/Aragon One/2019.md]:
```suggestion
  - Daniel: [GitHub](https://github.com/2color)
```

File: teams/Aragon One/2019.md (hunk 3 above)
Comment (luisivan) [reply teams/Aragon One/2019.md]: Makes sense, will point that out

File: teams/Aragon One/2019.md [hunk 5]
```diff
@@ -74,7 +71,7 @@ Improve the overall user experience of the platform and core apps by making the
 
 Focus areas for this initiative:
 
-- **Responsive UI**: Make the platform and core apps responsive so Aragon can be used from mobile browsers such as Status or Coinbase Wallet.
+- **✅ Responsive UI**: Make the platform and core apps responsive so Aragon can be used from mobile browsers such as Status or Coinbase Wallet.
 
 - **Native mobile app**: Build native (probably React Native makes the most sense) mobile Aragon clients so we can benefit from device and OS specific capabilities.
```
Comment (izqui) [inline teams/Aragon One/2019.md]: Should this be removed for now as well?

**Review by izqui — APPROVED** [review]

File: teams/Aragon One/2019.md [hunk 6]
```diff
@@ -64,19 +61,15 @@ Focus areas for this initiative:
...
-### I05 – Responsive experience
+### ✅ I05 – Responsive experience
 
 Improve the overall user experience of the platform and core apps by making the UI render well on any form factor, mobile wallet and browser, so users can seamlessly create, access and manage organizations.
 
-Focus areas for this initiative:
-
-- **Responsive UI**: Make the platform and core apps responsive so Aragon can be used from mobile browsers such as Status or Coinbase Wallet.
-
-- **Native mobile app**: Build native (probably React Native makes the most sense) mobile Aragon clients so we can benefit from device and OS specific capabilities.
```
Comment (luisivan) [inline teams/Aragon One/2019.md:79]: I don't think we have the bandwidth to do a native mobile app, given that there are higher priorities

Comment (luisivan) [reply teams/Aragon One/2019.md:79]: This would also require something like Frame Mobile

**Comment by luisivan** [comment]
Accidentally merged this into `master`, will unmerge now

**Comment by luisivan** [comment]
Created https://github.com/aragon/flock/issues/14 so this doesn't happen again

**Pull request merged**